cmd/snap/cmd_connect.go

```diff
@@ -27,8 +27,8 @@ import (
 
 type cmdConnect struct {
 	Positionals struct {
-		Offer SnapAndName `required:"yes"`
-		Use   SnapAndName
+		Use   SnapAndName `required:"yes"`
```

Comment:
How about `PlugSide` and `SlotSide` here? This would be consistent with the terms used in `client.Connect()`.